src/super_gradients/training/utils/detection_utils.py

```diff
@@ -170,9 +170,9 @@ def calc_bbox_iou_matrix(pred: torch.Tensor):
     return ious
 
 
-def change_bbox_bounds_for_image_size(boxes: np.ndarray, img_shape: Tuple[int, int]) -> np.ndarray:
+def change_bbox_bounds_for_image_size_inplace(boxes: np.ndarray, img_shape: Tuple[int, int]) -> np.ndarray:
```

Comment:
Although low chance, this might interfere with anyone expecting the old behaviour when calling the old method and expecting it to be inplace...
Perhaps a flag with the old behaviour as default would be preferable.

Reply:
Thanks! True, keeping a BC is a good suggestions. Fixed in fa50ec998ae65c2997a5f3dff3a6cbf729590827